client/db/src/lib.rs

```diff
@@ -996,6 +1005,8 @@ impl<T: Clone> FrozenForDuration<T> {
 /// Disk backend keeps data in a key-value store. In archive mode, trie nodes are kept from all
 /// blocks. Otherwise, trie nodes are kept only from some recent blocks.
 pub struct Backend<Block: BlockT> {
+	// Only needed for benchmarking.
+	db: Arc<dyn Database<DbHash>>,
```

Comment:
Should this field be feature gated with `#[cfg(feature = "runtime-benchmarks")]

Reply:
Or just access db from storage?

Reply:
I need to call the `commit` function on it which is not exposed by the Storage trait.  
But yes, there should be a feature flag.

Reply:
I mean remove the field and when you need to access it use self.storage.db instead of self.db (In backend it is not the storage trait but the actual struct)

Reply:
totally understandable :)